common/middleware/storeOasysSessionKey.test.js

```diff
@@ -0,0 +1,35 @@
+// Local dependencies
+const storeOasysSessionKey = require('./storeOasysSessionKey')
+
+describe('Put OASys session key information into session', () => {
+  let req
+  let res
+  beforeEach(() => {
+    req = {
+      query: {
+        sessionKey: '',
+      },
+      session: {
+        sessionKey: '',
+      },
```

Comment:
This isn't necessary and not a true reflection of a real request. The lines below should add it in if required 👍